platform-api/src/routers/auth.py

```
@@ -0,0 +1,44 @@
+        SELECT user.email 
+        FROM user 
+        JOIN session ON user.id = session.user_id
+        WHERE user.email = ? 
+          AND session.token = ? 
+          AND user.role IN ('sponsor') 
+          AND session.expires_at > (unixepoch() * 1000);
+    """
+
+    try:
+        result = await db.prepare(query).bind(username, password).first()
+    except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)